components/Shell.tsx

```diff
@@ -298,93 +298,88 @@ function UserDropdown({ small }: { small?: boolean }) {
   const query = useMeQuery();
   const user = query.data;
 
-  return (
-    !!user && (
-      <Dropdown>
-        <DropdownMenuTrigger asChild>
-          <div className="flex items-center space-x-2 cursor-pointer group">
-            <Avatar
-              imageSrc={user.avatar}
-              alt={user.username}
+  return user ? (
```

Comment:
not sure, did I fix this correctly? @KATT

Reply:
Yes you did :)